docs/source/usersguide/install.rst

```diff
@@ -113,34 +113,32 @@ Prerequisites
 
     * HDF5_ Library for portable binary output format
 
-      OpenMC uses HDF5 for binary output files. As such, you will need to have
-      HDF5 installed on your computer. The installed version will need to have
-      been compiled with the same compiler you intend to compile OpenMC with. If
-      you are using HDF5 in conjunction with MPI, we recommend that your HDF5
-      installation be built with parallel I/O features. An example of
-      configuring HDF5_ is listed below::
+      OpenMC uses HDF5 for many input/output files. As such, you will need to
+      have HDF5 installed on your computer. The installed version will need to
+      have been compiled with the same compiler you intend to compile OpenMC
+      with. On Debian derivatives, HDF5 and/or parallel HDF5 can be installed
```

Comment:
'If compiling with gcc from the repositories, users of Debian derivatives can...'